pkg/vadmin/add_node_at.go

```diff
@@ -57,7 +57,7 @@ func isLicenseLimitError(stdout string) bool {
 }
 
 // genAddNodeCommand returns the command to run to add nodes to the cluster.
-func (a Admintools) genAddNodeCommand(s *addnode.Parms) []string {
+func (a *Admintools) genAddNodeCommand(s *addnode.Parms) []string {
```

Comment:
I used a pointer receiver function. I am curious, any reason why you did not?

Reply:
We weren't changing anything in `Admintools` and it was a small struct. So a copy shouldn't be to bad. I'm okay with changing it to a pointer receiver. Especially if you are adding new fields to Admintools.